oauth2client/contrib/django_util/__init__.py

```diff
@@ -162,7 +162,7 @@ def test_callback(sender, request, credentials, **kwargs):
 
    oauth2_authorized.connect(test_callback)
 
-"""
+"""  # noqa: E501
```

Comment:
What is this? And is this obscure weird code the only way to express it?

Fun fact: three years ago I authored changes internal to Google that changed hundreds of thousands of lines of code in tens of thousands of files from `# pylint: disable-msg=W6403` (and the like) to `# pylint: disable=unused-variable`. Our codebase was better for it; tell me that the same is possible here?